features/user.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Managing users
+
+  Background:
+    Given I am logged in as an admin
+		And I am on the Dashboard
+
+  @javascript
+  Scenario: I can add a new user
```

Comment:
The second one yes, because selecting a submenu item has to be done differently when you are using a javascript enabled driver (because the submenu item is not immediately visible).